third_party/2and3/yaml/__init__.pyi

```diff
@@ -14,10 +14,10 @@ def scan(stream, Loader=...): ...
 def parse(stream, Loader=...): ...
 def compose(stream, Loader=...): ...
 def compose_all(stream, Loader=...): ...
-def load(stream, Loader=...): ...
-def load_all(stream, Loader=...): ...
-def safe_load(stream): ...
-def safe_load_all(stream): ...
+def load(stream, Loader=...): ...  # type: (Any, Loader) -> Any
```

Comment:
It seems weird to use comment-based annotations here. Why not just

```
def load(stream: Any, Loader: Loader = ...) -> Any: ...
```
?

Reply:
Also, for the stream, could something like `Union[str, IO[str]]` work?

Reply:
Updated.